### ISSUE TITLE
appindicator is not shown after instalation Gnome 3.26.0

### ISSUE THREAD

**Issue by denisok** [ubuntu/gnome-shell-extension-appindicator#87]
Hello,

I tried to install extension and it doesn't show up in the system panel.

Also I don't see any configuration icon for this extension. Does it have some?

Here is a log: 

```
Sep 18 15:39:06 linux-atrv org.gnome.Shell.desktop[4546]:  extracting: /home/dkondratenko/.local/share/gnome-shell/extensions/appindicatorsupport@rgcjonas.gmail.com/metadata.json
Sep 18 15:39:06 linux-atrv gnome-shell[4546]: Some code accessed the property 'ENABLED_EXTENSIONS_KEY' on the module 'extensionSystem'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: JS WARNING: [/home/dkondratenko/.local/share/gnome-shell/extensions/apps-menu@gnome-shell-extensions.gcampax.github.com/extension.js 65]: Too many arguments to method Clutter.Actor.add_child: expected 1, got 2
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: loading user theme: /usr/share/themes/Arc-Dark/gnome-shell/gnome-shell.css
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: System monitor applet enabling
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: System monitor applet enabling done
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: Some code accessed the property 'Logger' on the module 'util'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Sep 18 15:39:39 linux-atrv gnome-shell[4546]: [AppIndicatorSupport-DEBUG] Reload detected, destroying old watchdog
Sep 18 15:40:15 linux-atrv gnome-shell[4546]: Source ID 7048 was not found when attempting to remove it
```

Thanks,
Denis

**Comment by jhasse** [comment]
The extension doesn't have an icon by itself. You'll need to start a program like Remmina or Redshift-gtk and their indicators will appear in the top panel.

We also don't have any configuration options yet, sorry :(

**Comment by denisok** [comment]
I have Telegram and Hexchat

**Comment by denisok** [comment]
but you are right remmina shows up

**Comment by jhasse** [comment]
Which distribution are you using?

Hexchat doesn't work with indicators yet, see https://github.com/hexchat/hexchat/issues/1235.

**Comment by denisok** [comment]
openSUSE Tumbleweed

**Comment by jhasse** [comment]
And did you get Telegram from the official side? It might be possible to force it to use libappindicator by starting it with

```
env XDG_CURRENT_DESKTOP=Unity telegram
```

**Comment by denisok** [comment]
yes, Telegram is from official site.

> env XDG_CURRENT_DESKTOP=Unity telegram

didn't help

**Comment by jhasse** [comment]
Do you have libappindicator3.so in /usr/lib or /usr/lib64 ?

**Comment by denisok** [comment]
`ls -la /usr/lib64/libappindicator3*               
lrwxrwxrwx 1 root root    25 Sep 14 00:32 /usr/lib64/libappindicator3.so.1 -> libappindicator3.so.1.0.0
-rwxr-xr-x 1 root root 56464 Sep 14 00:32 /usr/lib64/libappindicator3.so.1.0.0                         

ls -la /usr/lib/libappindicator3*                                                                    
ls: cannot access '/usr/lib/libappindicator3*': No such file or directory                              
`

**Comment by jhasse** [comment]
Not sure why it isn't work then, sorry :(

I've created a new issue so we can track this.

**Comment by denisok** [comment]
Thanks!

**Comment by denisok** [comment]
BTW Skype 5.4.0.1 is also not there.

**Comment by jhasse** [comment]
I think that is https://github.com/electron/electron/issues/10427

**Comment by denisok** [comment]
I have installed fresh Tumbleweed on my home Laptop.

I can see telegram icon in notification area now.

The only problem, it is too small with my 4k monitor and scaling enabled:

![screenshot from 2017-09-19 08-54-18](https://user-images.githubusercontent.com/1289121/30579311-6f22c9ec-9d18-11e7-9402-c3e4c312f8b0.png)

**Comment by jhasse** [comment]
Hm ... I thought that was fixed with #72. Unfortunately I can't test it myself as I don't have a HiDPI screen and I'm still on GNOME 3.24 atm.
